### PR TITLE
[SMALLFIX]Inline documentation snippets

### PR DESCRIPTION
**Pull request by Hao-Kailong**

### CODE REVIEW

File: docs/cn/Clients-Alluxio-Java.md [hunk 1]
```diff
@@ -17,20 +17,37 @@ Alluxio通过文件系统接口提供对数据的访问。Alluxio中的文件提
 
 要使用Java代码获取Alluxio文件系统客户端，请使用：
 
-{% include File-System-API/get-fileSystem.md %}
+```java
+FileSystem fs = FileSystem.Factory.get();
```
Comment (yupeng9) [inline docs/cn/Clients-Alluxio-Java.md]: @Hao-Kailong Do we still need to keep the file `File-System-API/get-fileSystem.md`?

Comment (Hao-Kailong) [reply docs/cn/Clients-Alluxio-Java.md]: We don't have to

Comment (Hao-Kailong) [reply docs/cn/Clients-Alluxio-Java.md]: @yupeng9 So should I delete those files?

Comment (yupeng9) [reply docs/cn/Clients-Alluxio-Java.md]: yes, please remove the orphaned files.

**Comment by Hao-Kailong** [comment]
@gpang @yupeng9  I have removed the orphaned files. Please check again. Thank you~

**Pull request merged**